pkgs/applications/virtualization/singularity/packages.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, callPackage
+, fetchurl
+}:
+
+{
+  singularity-legacy = (callPackage ./template.nix rec {
+    pname = "singularity-legacy";
+    version = "3.8.6";
+    projectName = "singularity";
+
+    src = fetchurl {
+      url = "https://github.com/hpcng/singularity/releases/download/v${version}/singularity-${version}.tar.gz";
```

Comment:
`fetchFromGitHub`?

Reply:
That means switching from the vendored sources to the unvendored (vanilla) ones, which is also permitted after this PR.

I'll adopt the change if it is the preferred way of packaging.